lib/pointing_party/card.ex

```diff
@@ -5,18 +5,22 @@ defmodule PointingParty.Card do
   alias PointingParty.Repo
   alias PointingParty.Card
 
+  def points_range, do: [0, 1, 3, 5]
```

Comment:
It would make the module a little easier to read of all our function definitions were grouped together. Would you be opposed to moving this down somewhere below the call to the `schema` macro?